doc/manual.asciidoc

```diff
@@ -265,6 +265,11 @@ output files are out of date.
 rebuild those targets.
 _Available since Ninja 1.11._
 
+`multi-inputs`:: given a list of targets, print a list of all inputs used
+to rebuild those targets. Each line will constst of a target, a field separator
+and a file path. The list produced by the tool can be helpful if one
+would like to know which targets that are affected by a certain file.
```

Comment:
nit: Please clarify why this is different from `inputs` here. In particular explain that this prints different sets of inputs, one per target, and that a given input may appear several times in the output if it is used by more than one target. Maybe add one example as you did in your commit message for clarity?

Also s/constst/consist/ :-)